data.sql

```diff
@@ -1,2 +1,3 @@
 INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered,weight_kg) VALUES (1, 'Agumon', '02/03/20', 0, true, 10.23),(2, 'Gabumon', '11/15/18', 2, true, 8.0), (3, 'Pikachu', '01/7/21', 1,false,15.04), (4, 'Davimon', '05/12/17', 5,true,11);
 
+INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered,weight_kg) VALUES (5, 'Charmander', '02/08/20', 0, false, -11),(6, 'Plantmon', '11/15/21', 2, true, -5.7), (7, 'Squirtle', '04/02/93',3 , false, -12.13), (8, 'Angemon', '06/12/05', 1, true, -45),(9, 'Boarmon', '06/07/05', 7, true, 20.4),(10, 'Blossom', '10/13/98', 3, true, 17),(11, 'Ditto', '05/14/22', 4, true, 22);
```

Comment:
- [ ] I got this error message while trying to populate your table with the data inside the data.sql. Kindly look into it and fix.

### Error Message
![postgres](https://user-images.githubusercontent.com/26597432/235776734-c2ebd618-4b0f-432e-8e2b-8c47a58806a9.png)

Reply:
@Qoosim Thanks for the review and your time 👍 . It was working on my side but I have made the changes and hope this time it gets accepted. 🤞 
Thanks Again. 💮 
![Screenshot (72)](https://user-images.githubusercontent.com/117263981/235844524-9ea2da83-a7d2-4130-be8a-842db1aeb9d0.png)